bcda/api.go

```diff
@@ -137,6 +149,31 @@ func jobStatus(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func createOperationOutcome(outcome string) []byte {
```

Comment:
Should this function be developed such that all the fields can be set?  It looks like we are only supporting the ability to set `Id`, but would we ever want the ability to set severity, code, diagnostics, etc, etc, etc (rather than leave them as hardcoded empty strings)?

Reply:
True. We absolutely want to set the real details of the error response once we start processing for errors. Currently nothing really causes a failure for a job because we're just serving up a static file, but you are correct in the future we will want this info to be populated. This makes me think of a future design question: I think this method will change from passing function args to just handling of the db job object.